workflow/workflow_test.go

```diff
@@ -308,6 +308,73 @@ func TestWithDoubleSerializationDetection(t *testing.T) {
 	assert.Equals(t, errorDetect.SerializeCnt+errorDetect.UnserializeCnt > 0, true)
 }
 
+var simpleValidLiteralInputWaitWorkflowWithDefaults = `
+version: v0.2.0
+input:
+  root: RootObject
+  objects:
+    RootObject:
+      id: RootObject
+      properties: 
+        wait_time_def:
+          type:
+            type_id: integer
+          default: 1
+          required: false
```

Comment:
This isn't really germane to this test:  the double-operation checking is in the `"error_detector"` property's schema, so _that's_ what we need to be setting defaults on (via the second-to-last parameter in the call to `schema.NewPropertySchema()` at line 354 , I think).

So, I'm not sure that we need a new test case (`Test_DoubleSerializationDetection_Default()`) -- I think we can do it all with `TestWithDoubleSerializationDetection()` by varying what we pass to `schema.NewPropertySchema()` and to `preparedWorkflow.Execute()`.

Reply:
removed